app/services/data_migrations/backfill_feedback_form_complete.rb

```diff
@@ -0,0 +1,19 @@
+module DataMigrations
+  class BackfillFeedbackFormComplete
```

Comment:
PR looks good chris – do you mind moving the migration out into a separate (follow up PR) and then I think we're good to go!